.github/workflows/test-common.yml

```diff
@@ -65,7 +48,19 @@ jobs:
           set -x # print commands that are executed
           coverage erase
 
-          ./scripts/runtox.sh "py${{ matrix.python-version }}$" --cov=tests --cov=sentry_sdk --cov-report= --cov-branch --ignore=tests/integrations
+          ./scripts/runtox.sh "py${{ matrix.python-version }}-${{ matrix.variant }}$" --cov=tests --cov=sentry_sdk --cov-report= --cov-branch --ignore=tests/integrations
           coverage combine .coverage*
           coverage xml -i
           codecov --file coverage.xml
+
+  check_required_tests:
+    name: All common tests passed or skipped
+    needs: test
+    # Always run this, even if a dependent job failed
+    if: always()
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check for failures
+        if: contains(needs.test.result, 'failure')
+        run: |
+          echo "One of the dependent jobs have failed. You may need to re-run it." && exit 1
```

Comment:
I don't have permission to do this but if we could change the required check to this step like for all the integration tests, that'd be great.

Reply:
For testing purposes, I have reverted your naming change of the common tests. I hope you do not mind if I am committing to your branch.

Reply:
resolve, because it is outdated now.